packages/cdk/.npmrc

```diff
@@ -0,0 +1,2 @@
+engine-strict=true
+package-lock=false
```

Comment:
Since we are using shrinkwrap.json, I disable producing package-lock.json here

Reply:
I don't think this is doing what you think it is. Setting this to false will also ignore the shrinkwrap and effectively bump our dependencies on every install.

Reply:
You are correct.